protocol/shannon/observation.go

```diff
@@ -166,6 +168,12 @@ func buildEndpointObservation(
 	observation.Supplier = endpoint.supplier
 	observation.EndpointUrl = endpoint.url
 
+	// Add endpoint response details if not nil (i.e. success)
+	if endpointResponse != nil {
+		*observation.EndpointResponseHttpStatusCode = int32(endpointResponse.HTTPStatusCode)
```

Comment:
I believe this is the HTTP status code returned by the backend service, relayed to PATH via RelayMiner and [deserialized via the SDK](https://github.com/pokt-network/shannon-sdk/blob/e527d4172770cba5606678fdd381b5481e124788/types/response.go#L51)
The HTTP status code from the call to RelayMiner is ignored in `sendHttpRelay`.

Just making sure this is what we intend to capture here. If so, I suggest a different name for the field to clarify this point.